src/main/java/pro/sky/homework_2_9/DepartmentController.java

```diff
@@ -9,37 +9,38 @@
 @RequestMapping("/departments")
 public class DepartmentController {
 
-    private final DepartmentServiceC departmentService;
+    private final DepartmentServiceImpl departmentService;
```

Comment:
```suggestion
    private final DepartmentService departmentService;
```
инъекцию всегда надо делать по интерфейсу, а не реализации